cpp/include/cpu_operations.h

```diff
@@ -62,6 +76,20 @@ class CpuOperations {
         return (a.array() + scalar);
     }
   }
+  /// This is a function that adds two matricies and returns the resulting
+  /// matrix.
+  ///
+  /// \param a
+  /// Input Matrix 1
+  /// \param b
+  /// Input Matrix 2
+  ///
+  /// \return
+  /// This function returns a matrix that is the resulatant of adding two
+  /// input matricies.
+  ///
+  /// \sa
+  /// \ref Add
```

Comment:
Same thing here, it would be great if we can have two Add functions refer to each other.